tools/run_tests/run_tests.py

```diff
@@ -445,6 +445,10 @@ def _get_tox_envs(self, compiler):
       return ('py27',)
     elif compiler == 'python3.4':
       return ('py34',)
+    elif compiler == 'python3.5':
+      return ('py35',)
+    elif compiler == 'python3.6':
+      return ('py36',)
     else:
       raise Exception('Compiler %s not supported.' % compiler)
 
```

Comment:
In line 835 there is a list of supported compilers, I think you might need to add python3.5 and python3.6 there as well.